frontend/pages/create-project.tsx

```diff
@@ -0,0 +1,221 @@
+import React, {useState, useEffect} from 'react';
+import Link from 'next/link';
+import { PhotoIcon } from '@heroicons/react/24/solid';
+import { useFormik } from 'formik';
+
+interface CreateProjectProps {}
+
+const CreateProject: React.FC<CreateProjectProps> = () => {
+    const maxCharacterLimit = 20;
```

Comment:
Do declare this as a variable, just send it as a props to the textarea
Make this number bigger